### PR TITLE
Use hrtime in JS uptimeMillis when running in NodeJS

### LINKED ISSUE #103: uptimeMillis() fails in JS tests
`ReferenceError: window is not defined`. Should use `Date.now()`.

### PR DESCRIPTION
**Pull request by arkivanov**
Fixes #103

### CODE REVIEW

File: reaktive/src/jsMain/kotlin/com/badoo/reaktive/utils/UptimeMillis.kt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package com.badoo.reaktive.utils
 
-import kotlin.browser.window
+import kotlin.js.Date
 
 internal actual val uptimeMillis: Long
-    get() = window.performance.now().toLong()
+    get() = Date.now().toLong()
```
Comment (lukaville) [inline reaktive/src/jsMain/kotlin/com/badoo/reaktive/utils/UptimeMillis.kt]: What's the problem with `window.performance.now`? Looks like it's available in all modern browsers https://caniuse.com/#search=performance.now
If we use Date then uptimeMillis will not be monotonic.

Comment (CherryPerry) [reply reaktive/src/jsMain/kotlin/com/badoo/reaktive/utils/UptimeMillis.kt]: But not in NodeJS as I can see.
https://stackoverflow.com/questions/23003252/window-performance-now-equivalent-in-nodejs

Comment (CherryPerry) [reply reaktive/src/jsMain/kotlin/com/badoo/reaktive/utils/UptimeMillis.kt]: We use NodeJS to run our tests.

Comment (lukaville) [reply reaktive/src/jsMain/kotlin/com/badoo/reaktive/utils/UptimeMillis.kt]: Let's use https://nodejs.org/api/process.html#process_process_hrtime_time for nodejs then and window.performance.now for browsers

**Comment by arkivanov** [comment]
How we can distinguish?

**Comment by lukaville** [comment]
@arkivanov 
We can copy-paste from Klock or add it as a dependency https://github.com/korlibs/klock/blob/master/klock/src/jsMain/kotlin/com/soywiz/klock/internal/KlockInternal.kt

**Comment by arkivanov** [comment]
Makes sense, why can't we always use hrtime?

**Comment by lukaville** [comment]
> Makes sense, why can't we always use hrtime?

hrtime is not available in browsers

**Comment by arkivanov** [comment]
Last question, what exactly wrong with Date?

**Comment by lukaville** [comment]
> Last question, what exactly wrong with Date?

It may decrease during the execution of the program and it can lead to various unwanted side effects and broken logic especially in time-related operators

File: reaktive/src/jsMain/kotlin/com/badoo/reaktive/utils/UptimeMillis.kt (hunk 1 above)
Comment (arkivanov) [reply reaktive/src/jsMain/kotlin/com/badoo/reaktive/utils/UptimeMillis.kt]: Fixed pls check

**Pull request merged**